codebuild/bin/utils.sh

```diff
@@ -24,10 +24,19 @@ get_latest_release(){
 
 gh_login(){
     # Takes secrets manager key as an argument
-    aws secretsmanager get-secret-value --secret-id "$1" --query 'SecretString' --output text |jq -r '.secret_key'| gh auth login --with-token
+    # This GH personal access token must have 'repo' permsissions to work.
+    gh auth status || aws secretsmanager get-secret-value --secret-id "$1" --query 'SecretString' --output text |jq -r '.secret_key'| gh auth login --with-token
+
     #gh auth status
 }
 
+criterion_install_deps(){
+    make install
+    source "$HOME"/.cargo/env
```

Comment:
Why is this source command required?

Reply:
Rust might just have been installed, in the CI context wanted to ensure cargo is in the path.

Reply:
Oh I see, .cargo/env adds .cargo/bin to the path. Might be worth a comment, but maybe it's more standard than I think.